maintainers/maintainer-list.nix

```diff
@@ -7019,4 +7019,9 @@
     githubId = 1986844;
     name = "Daniel Wheeler";
   };
+  rakesh4g = {
+    email = "rakeshgupta4u@gmail.com";
+    github = "Rakesh4G";
+    name = "Rakesh Gupta";
+  };
```

Comment:
please add githubId and make this into a seperate commit

Reply:
Not needed now.